### PR TITLE
feat: Migrate design system components import to design-system repo - I

### LINKED ISSUE #15518: [Task]: Import changes to Scroll Indicator
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

_No response_

### PR DESCRIPTION
**Pull request by albinAppsmith**
## Description

This PR has changed the import of following components to design-system repository.

- Icon
- Button
- Breadcrumbs
- EmojiPicker
- EmojiReaction
- AppIcon
- Menu Item
- Icon Selector
- Scroll Indicator fixes #15518
- Text Input fixes https://github.com/appsmithorg/appsmith/issues/15516
- Label with Tooltip fixes https://github.com/appsmithorg/appsmith/issues/15515
- Checkbox fixes https://github.com/appsmithorg/appsmith/issues/15492
- Dropdown fixes https://github.com/appsmithorg/appsmith/issues/16000

Fixes [#15561](https://github.com/appsmithorg/appsmith/issues/15561)

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- Tested in local

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/components/ads/Button.test.tsx [hunk 1]
```diff
@@ -1,56 +0,0 @@
-import React from "react";
```
Comment (tanvibhakta) [inline app/client/src/components/ads/Button.test.tsx:1]: This file is missing in the ds repo, can you please add it?

Comment (albinAppsmith) [reply app/client/src/components/ads/Button.test.tsx:1]: Okay

**Review by tanvibhakta — APPROVED** [review]

**Review by tanvibhakta — APPROVED** [review]

**Review by tanvibhakta — DISMISSED** [review]

**Review by tanvibhakta — DISMISSED** [review]

**Review by tanvibhakta — DISMISSED** [review]

**Comment by shadabbuchh** [comment]
Below are the issues observed in this PR:

1. Checkbox Component:

- Checkbox for any custom column, save/discard column & all columns other than Plain text & number is not appropriate.


2. Button Component:

- The colour of the Chart widget - Add Series button is darker in DP when compared to prod/release.


@albinAppsmith, @tanvibhakta

**Comment by shadabbuchh** [comment]
Issues in the Dropdown component:

- The indentation for options in a dropdown is not appropriate. Please note that this issue is present in all impact areas of the Dropdown component.


Cc: @albinAppsmith, @tanvibhakta

**Comment by shadabbuchh** [comment]
Retested the issues. Everything working as expected.

**Review by tanvibhakta — APPROVED** [review]

**Pull request merged**